lib/tasks/one_off/undefined_area_fix.rake

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+namespace :one_off do
+  desc "Fix areas for manually added site addresses (see RUBY-3013)"
+  task undefined_area_fix: :environment do
+    # 1. Find registrations with undefined EA area
+    regs = find_registrations_with_undefined_area
+
+    stats = { regs_count: regs.count, addr_fixed_count: 0, addr_partcode_fixed_count: 0, addr_not_fixed_count: 0 }
+
+    # 2. Find out the number of site addresses that can be fixed by matching cobtact and operator addresses
```

Comment:
Minor: Typo (contact)

Reply:
that comment is no longer relevant, I've removed it